app/controllers/jobs_controller.rb

```diff
@@ -1,15 +1,33 @@
 class JobsController < ApplicationController
+
   def index
     @company = Company.find(params[:company_id])
     @jobs = @company.jobs
+    @contact = Contact.new
+    @contacts = Contact.all
```

Comment:
do i want to see all contacts on the `company_jobs show`? i just want to see the contacts for this company.